utils/fstack.c

```diff
@@ -1906,7 +1906,6 @@ static int __read_rstack(struct ftrace_file_handle *handle,
 
 	if (has_perf_data(handle)) {
 		p = read_perf_data(handle);
-		perf = &handle->perf[p];
```

Comment:
That's ok as long as we don't access the variable.

Reply:
Aha, Thank you for your review.
I will close PR.